h2o-automl/src/main/java/ai/h2o/automl/Leaderboard.java

```diff
@@ -653,7 +652,7 @@ public static final TwoDimTable makeTwoDimTable(String tableHeader, String sort_
       return new TwoDimTable(tableHeader,
               "no models in this leaderboard",
               rowHeaders,
-              Leaderboard.colHeaders(sort_metric, other_metrics),
```

Comment:
Is this intended? How does it relate to this PR?

Reply:
Yes, it is intended just to put the point across no leaderboard is binomial.